tests/test_messages.py

```diff
@@ -54,7 +70,8 @@ def test_deserialize_with_required_type():
     deserialized_message = Message.deserialize(message, FeeInfo)
     assert isinstance(deserialized_message, FeeInfo)
 
-    # during deseriaisation the `message_type`is removed, add it back
+    # during deserialization the `message_type`is removed, add it back
```

Comment:
nice catch. please add a space after `message_type` as well and remove the newline.